### PR TITLE
Use service theme

### PR DESCRIPTION
**Pull request by stefanalund**
## Hey, I just made a Pull Request!

Screenshot 2020-06-08 at 14 29 53

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/core/src/layout/Header/Header.tsx [hunk 1]
```diff
@@ -104,16 +104,11 @@ const TypeFragment: FC<TypeFragmentProps> = ({ type, typeLink, classes }) => {
   }
 
   if (!typeLink) {
-    return (
-      // </Link>
-      <Typography className={classes.type}>{type}</Typography>
-    );
+    // TODO: Add breadcrumps.
```
Comment (benjdlambert) [inline packages/core/src/layout/Header/Header.tsx:107]: breadcrumbs*

Comment (stefanalund) [reply packages/core/src/layout/Header/Header.tsx:107]: Some serious nitpicking going on here 😬

Comment (Rugvip) [reply packages/core/src/layout/Header/Header.tsx]: You say that now, but when we get a PR adding breadcrumps you won't be laughing!

Comment (stefanalund) [reply packages/core/src/layout/Header/Header.tsx]: Now I'm laughing at least! 😄

**Review by freben — APPROVED** [review]
Nice but... Does your screenshot have no gradients in the header?

**Comment by stefanalund** [comment]
@katz95 is making another pass at the colors and gradients. Let's adress in a follow-up PR once finalised.

**Pull request merged**